diff --git a/dist/esm/src/debug/http-debug.controller.js b/dist/esm/src/debug/http-debug.controller.js
index 65f5ef36e3226bb22359a6dc4c055e65b94ef084..5299f7f05b21a6c60e07d9be93490737990d93be 100644
--- a/dist/esm/src/debug/http-debug.controller.js
+++ b/dist/esm/src/debug/http-debug.controller.js
@@ -9,13 +9,17 @@
  */
 import { registerStaticHttpController } from '@deepkit/http';
 import { AppModule, findParentPath } from '@deepkit/app';
+import { dirname } from 'path';
+import { getCurrentFileName } from '@deepkit/core';
+
 export function registerDebugHttpController(module, path) {
-    const localPath = findParentPath('node_modules/@deepkit/framework-debug-gui/dist/framework-debug-gui', __dirname);
+    const currentDir = dirname(getCurrentFileName());
+    const localPath = findParentPath('node_modules/@deepkit/framework-debug-gui/dist/framework-debug-gui', currentDir);
     if (localPath) {
         registerStaticHttpController(module, { path, localPath, groups: ['app-static'], controllerName: 'FrameworkDebuggerController' });
     }
     else {
-        console.log('Warning: node_modules/@deepkit/framework-debug-gui no build found in ' + __dirname);
+        console.log('Warning: node_modules/@deepkit/framework-debug-gui no build found in ' + currentDir);
     }
 }
 registerDebugHttpController.__type = [() => AppModule, 'module', 'path', 'registerDebugHttpController', 'PP"7!2"&2#$/$'];
