diff --git a/dist/esm/src/stopwatch.js b/dist/esm/src/stopwatch.js
index 4053c1c4131abd93b1d9f5e6f9de6495bb8b320d..d4a7798e7de04443d2b54ea983cf1d8b9e97fee7 100644
--- a/dist/esm/src/stopwatch.js
+++ b/dist/esm/src/stopwatch.js
@@ -101,8 +101,8 @@ export class Stopwatch {
             if (!zone)
                 return new NoopStopwatchFrame();
             context = zone.stopwatchContextId;
-            if (!context)
-                throw new Error('No Stopwatch context given');
+            //if (!context)
+            //    throw new Error('No Stopwatch context given');
         }
         const id = ++frameId;
         const worker = this.store.add({
